gum/guminterceptor.c

```diff
@@ -901,6 +923,23 @@ _gum_function_context_on_leave (FunctionContext * function_ctx,
 #endif
 }
 
+#ifdef HAVE_QNX
+__attribute__ ((naked))
+static void
+_gum_exec_callback_func_with_side_stack (GumInvocationListener * listener_instance,
+                               GumInvocationContext * invocation_ctx,
```

Comment:
The arguments should be visually aligned. Also, we only underscore-prefix internal functions that are non-static.